portfolio/src/main/webapp/style.css

```
@@ -1,9 +1,85 @@
 #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+  margin: 20px;
 }
+#nav_bar{
+    display: flex;
+    flex-direction: row;
+}
+#scroll_buttons{
+    margin-left: auto;
+}
+.flex_hor{
```

[4.1.4](https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters): Separate words in class names by a hyphen.
```suggestion
.flex-hor {
```